projects/angular/src/forms/common/wrapped-control.spec.ts

```diff
@@ -444,6 +455,19 @@ export default function (): void {
         expect(this.input.getAttribute('aria-describedby')).toContain('-success');
       }));
 
+      it('adds the aria-describedby with helper and success ids', fakeAsync(function (this: TestContext) {
+        setupTest(this, WithControlAndSuccess, TestControl3);
+        this.input.focus();
+        this.fixture.componentInstance.model = 'test';
+        this.input.blur();
+        this.fixture.detectChanges();
+        tick();
+
+        const describedByIds: string[] = this.input.getAttribute('aria-describedby').split(' ');
+        const expectedIds = [`${this.input.id}-helper`, `${this.input.id}-success`];
+        expect(describedByIds).toEqual(expectedIds);
```

Comment:
```suggestion
        expect(this.input.getAttribute('aria-describedby')).toEqual(`${this.input.id}-helper ${this.input.id}-success`);
```

Why not this? It's less code and more readable IMO.